storey/flow.py

```diff
@@ -1464,6 +1505,7 @@ class ParallelExecutionRunnable:
         block the main event loop thread.
     * "asyncio" – To run in an asyncio task. This is appropriate for I/O tasks that use asyncio, allowing the event
         loop to continue running while waiting for a response.
+    * "shared_proxy" - To run the shared model of the graph.
```

Comment:
Should probably elaborate further and avoid mention of model here (since this code is generic and not specific to model serving).